main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
This while loop would not loop the required times (10). Consider using a for loop instead since we know how many times we want to iterate.